components/input/input.css

```diff
@@ -0,0 +1,61 @@
+:root {
+	--error-color: #dc3545;
+	--success-color: #1b7052;
+}
```

Comment:
You can add these variables in a seperate base file.

Reply:
Done all the changes you have suugested. Thank you for suggestion